module-4/backend/week_three.md

```diff
@@ -5,15 +5,45 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+
+Node allows you to run javascript outside of the browser. It uses the v8 engine but adds additional functionality that is not available in the browser so that you can use it for a backend.
+
 2. What is Express? What is Express similar to in the Ruby world?
+
+Express is built on top of node and is like Rails in the ruby world. It gives you a framework to use javascript to write server side code. It takes things that node has and makes them easier to work with similar to what jquery does for client side javascript.
+
 3. How do we setup a route when creating an API with Node and Express? Please provide a code snippet.
+
+`app.post('/api/v1/foods', FoodsController.postFood)`
```

Comment:
I would write out the function here or at least the parameters it takes rather thank just a function handler